### PR TITLE
Advertise preorder links

### PR DESCRIPTION
**Pull request by pavpanchekha**
This PR adds a block to the main page asking people to preorder, and subtler calls to the sidebar as well.

Desktop style, main page:

image

Mobile style, main page:

image

Desktop style, chapter:

imageScreenshot 2024-07-17 at 5 36 57 PM

Mobile style, chapter

Screenshot 2024-07-17 at 5 49 36 PM

### CODE REVIEW

File: book/index.md [hunk 1]
```diff
@@ -14,9 +14,21 @@ chapters make inaccurate claims, fail to match published chapters, or
...
 :::
 
-::: {.signup}
+::::: {.wide-ad}
+![The cover for Web Browser Engineering, from Oxford University Press](im/cover.jpg)
+
+::: {.description}
+# Pre-order Web Browser Engineering
+
+_Web Browser Engineering_ is will be published by Oxford University
```
Comment (chrishtr) [inline book/index.md:23]:
```suggestion
_Web Browser Engineering_ will be published by Oxford University
```

File: book/index.md [hunk 2]
```diff
@@ -14,9 +14,21 @@ chapters make inaccurate claims, fail to match published chapters, or
...
+Press before the end of the year. **Pre-order now** to get it as soon as
+possible.
+
+[Preorder now](https://global.oup.com/academic/product/web-browser-engineering-9780198913856?cc=gb){.go-link}
```
Comment (chrishtr) [inline book/index.md:27]:
```suggestion
```

File: book/index.md [hunk 3]
```diff
@@ -14,9 +14,21 @@ chapters make inaccurate claims, fail to match published chapters, or
...
 
-::: {.signup}
+::::: {.wide-ad}
+![The cover for Web Browser Engineering, from Oxford University Press](im/cover.jpg)
+
+::: {.description}
+# Pre-order Web Browser Engineering
+
+_Web Browser Engineering_ is will be published by Oxford University
+Press before the end of the year. **Pre-order now** to get it as soon as
```
Comment (chrishtr) [inline book/index.md:24]:
```suggestion
Press before the end of the year. [Pre-order now](https://global.oup.com/academic/product/web-browser-engineering-9780198913856?cc=gb){.go-link} to get it as soon as
```

File: book/index.md [hunk 4]
```diff
@@ -14,9 +14,21 @@ chapters make inaccurate claims, fail to match published chapters, or
...
+possible.
```
Comment (chrishtr) [inline book/index.md:25]:
```suggestion
it comes out!
```

File: www/book.css [hunk 5]
```diff
@@ -301,6 +301,32 @@ form .checkoff, form button[type=submit] { margin-top: 1em; }
 #signup-close:active { color: red; }
 #signup-close::before { font-size: 24px; content: "❌︎"; }
 
+.wide-ad { display: flex; gap: 1em; padding: 1em; border: 2px solid #59b6ce; }
```
Comment (chrishtr) [inline www/book.css:304]: How does it look in dark mode?

**Comment by pavpanchekha** [comment]
Ach, I wasn't done yet, I shouldn't have requested review. Soon!

**Comment by pavpanchekha** [comment]
Dark mode, main page:

image

Dark mode, desktop, chapter:

imageimage

Dark mode, mobile, chapter:

image

**Comment by pavpanchekha** [comment]
Ok, way better in dark mode with a border:

imageimage

**Comment by pavpanchekha** [comment]
The actually-important discussion is that I am linking to the paperback, not the hardback. I don't know if you have thoughts on this, Chris, but it's almost certainly the most impactful decision in this PR :)

**Comment by pavpanchekha** [comment]
I just don't think people will want to pay $180 for WBE, but they might pay $50.

File: book/index.md [hunk 6]
```diff
@@ -14,9 +14,19 @@ chapters make inaccurate claims, fail to match published chapters, or
...
-::: {.signup}
+::::: {.wide-ad}
+![The cover for Web Browser Engineering, from Oxford University Press](im/cover.jpg)
+
+::: {.description}
+# Pre-order _Web Browser Engineering_
+
+_Web Browser Engineering_ will be published by Oxford University
+Press before the end of the year. To get it as soon as it's out,
+[pre-order now!](https://global.oup.com/academic/product/web-browser-engineering-9780198913863)
```
Comment (chrishtr) [inline book/index.md:25]: Linking to the paperback makes more sense--a much better price.

**Review by chrishtr — APPROVED** [review]

**Pull request merged**